### ISSUE TITLE
Проблема с Local Tunnel

### ISSUE THREAD

**Issue by TIIGRUS** [tars/tars#412]
На Win10 падает туннель, то есть не расшаривает и даже не запускает. И даже если удалось запустить, то через какое-то время тунель снова падает и нужно перезагружать. 

Возможно ли прикрутить [ngrok](https://ngrok.com/) в качестве расшаривание верстки?

**Comment by artem-malko** [comment]
@TIIGRUS 
Подскажи, какая версия tars и tars-cli используется?
Какой терминал используется?

**Comment by TIIGRUS** [comment]
> Подскажи, какая версия tars и tars-cli используется?
> Какой терминал используется?

Win 10 - система. 
Терминал использую встроенный в Visual Studio Code, он удобней, и если не ошибаюсь, то это PowerShell терминал.
На момент падания терминала версия(TARS-CLI version: "1.11.1").
Скрин ошибки, сейчас падает с ошибкой сертификата. 
Ngrok работает норм без падения. 

![image](https://user-images.githubusercontent.com/11195373/72249864-fbc44580-3602-11ea-9dd1-1f50f063f390.png)

**Comment by TIIGRUS** [comment]
> Ссылка на ngrok не работает(

https://ngrok.com

**Comment by TIIGRUS** [comment]
Отключал антивир, Защитник Win10, не помогает либо временно.

**Comment by artem-malko** [comment]
Понял, пожалуй можно попробовать)

**Comment by TIIGRUS** [comment]
> Понял, пожалуй можно попробовать)

Спасибо, было бы не плохо, так как проблема не только у меня одного.)

**Comment by bukhalo** [comment]
> Возможно ли прикрутить ngrok в качестве расшаривание верстки?

Можно, но зачем? ngrok это стороннее ПО не имеющее никакого отношения к миру JS. Что мешает добавить скрипт запуска ngrok в package.json?

```json
"scripts": {
  "tunnel": "ngrok http 300",
}
```

> Скрин ошибки, сейчас падает с ошибкой сертификата.
> ![image](https://user-images.githubusercontent.com/11195373/72249864-fbc44580-3602-11ea-9dd1-1f50f063f390.png)

Всё верно, у https://localtunnel.me итёк срок действия сертификата

**Comment by TIIGRUS** [comment]
> Что мешает добавить скрипт запуска ngrok в package.json?
Попробую, спасибо за идею.

**Comment by TIIGRUS** [comment]
Сори, случайно закрыл.

**Comment by artem-malko** [comment]
@TIIGRUS @bukhalo идея про script крутая, кстати)

**Comment by TIIGRUS** [comment]
> Что мешает добавить скрипт запуска ngrok в package.json?

Я понимаю ,что тут все профи и наверное у всех все стартует с первого раза, но сожалению у меня на завелся выше приведенный метод.

**Comment by bukhalo** [comment]
> Я понимаю ,что тут все профи и наверное у всех все стартует с первого раза, но сожалению у меня на завелся выше приведенный метод. )))

Только что попробовал с чистым проектом – всё ок. Можете вставить готовые скрипты в `package.json` своего проекта:
```json
"scripts": {
  "dev": "tars dev -l & npm run tunnel",
  "tunnel": "ngrok http 3004",
  "test": "echo \"Error: no test specified\" && exit 1"
},
```
Ну и запускать через `npm run dev` соответственно.

**Comment by artem-malko** [comment]
Видимо метод не завелся из-за того, что порт был 3000 указан, а tars 3004 запускает.

**Comment by TIIGRUS** [comment]
@bukhalo Да, вроде получилось. Правда приходится запускать два экземпляра консоли. А я почему-то решил, что оно все будет запускаться по одной команде. Возможно поэтому в прошлый раз показалось, что не завелось.
Или я что-то не так делаю?)

![image](https://user-images.githubusercontent.com/11195373/72847654-b1e6f980-3cab-11ea-9386-2017499ff332.png)

**Comment by TIIGRUS** [comment]
@artem-malko @bukhalo 
Спасибо, получилось не плохо. Юзаю уже несколько недель, неплохая альтернатива LocalTunell.
